test/unit/math/prim/functor/ode_rk45_prim_test.cpp

```diff
@@ -1,68 +1,10 @@
 #include <stan/math/prim.hpp>
 #include <gtest/gtest.h>
 #include <test/unit/util.hpp>
+#include <test/unit/math/prim/functor/ode_test_functors.hpp>
```

Comment:
All the changes in the ODE files were an alternate fix to a problem that popped up here: https://github.com/stan-dev/math/pull/1993#issuecomment-667664487

They don't change behavior. They just rearrange test code a bit so that the jumbo tests (#1965) build correctly.